web/src/components/UploadBooksDropdown.module.css

```diff
@@ -0,0 +1,61 @@
+.dropdown {
+    composes: body3 from '../App.css';
+    width: 310px;
+    height: 64px;
+    background-color: white;
+    outline: none;
+    border-radius: 15px;
+    cursor: pointer;
+}
+
+.dropdownElement {
+    composes: body3 from '../App.css';
+    background-color: white;
+    border-color: black;
+    border-width: 1px;
+    width: 310px;
+    height: 60px;
+    text-align: left;
+    cursor: pointer;
+}
+
+.firstDropdownElement {
+    composes: dropdownElement;
+    border-radius: 10px 10px 0 0;
+}
+
+.lastDropdownElement {
+    composes: dropdownElement;
+    border-radius: 0 0 10px 10px;
+}
```

Comment:
This is web so we actually have fully featured CSS!!! Can you use :first/last-of-type to combine these and get rid of the annoying index logic?

https://www.w3schools.com/cssref/sel_first-of-type.asp
https://www.w3schools.com/cssref/sel_last-of-type.asp